src/test/migrations/steps/20240925193041-step-fix-growing-stock-composition-2025.ts

```diff
@@ -0,0 +1,41 @@
+import { AssessmentNames } from 'meta/assessment'
+
+import { AssessmentController } from 'server/controller/assessment'
+import { BaseProtocol, Schemas } from 'server/db'
+
+export default async (client: BaseProtocol) => {
+  const { assessment, cycle } = await AssessmentController.getOneWithCycle(
+    { assessmentName: AssessmentNames.fra, cycleName: '2025' },
+    client
+  )
+
+  const schemaAssessment = Schemas.getName(assessment)
+
+  await client.query(`
+    update ${schemaAssessment}."table" t
+    set props = jsonb_set(
+        jsonb_set(
+          jsonb_set(
+            props,
+            '{columnNames,${cycle.uuid}}',
+            '["scientific_name", "common_name", "growingStockMillionCubicMeter", "growingStockPercent"]'::jsonb
+          ),
+          '{columnsExport,${cycle.uuid}}',
+          '["scientific_name", "common_name", "growingStockMillionCubicMeter", "growingStockPercent"]'::jsonb
+        ),
+        '{columnsExportAlways,${cycle.uuid}}',
+        '[]'::jsonb
+      )
+    from (
+      select t2.id
+      from ${schemaAssessment}."table" t2
+      left join ${schemaAssessment}.table_section ts on t2.table_section_id = ts.id
+      left join ${schemaAssessment}.section s on s.id = ts.section_id
+      where s.props ->> 'name' = 'growingStockComposition'
+      and t2.props ->> 'name' = 'growingStockComposition2025'
+    ) as src
+    where t.id = src.id;
+  `)
+
+  await AssessmentController.generateMetadataCache({ assessment }, client)
+}
```

Comment:
For some reason, after I checked out to the latest to the latest development-next I wasn't able to see the data in the updated table:
![image](https://github.com/user-attachments/assets/85f411be-e55e-44e6-9791-dfa648585871)

Before I was able to with the new fixes 🤔 
This is a file I generated before:
![image](https://github.com/user-attachments/assets/e77d0173-ed95-48b8-b5b8-a22850896fcc)